src/ClientGenerator/CSharpCodeGenerator.cs

```diff
@@ -160,7 +162,7 @@ protected override void AddCreateObjectReferenceMethods(GrainInterfaceData grain
             var invokerField = new CodeSnippetTypeMember(fieldImpl);
             factoryClass.Members.Add(invokerField);
 
-            var methodImpl = String.Format(@"
```

Comment:
If the codebase primarily uses `String.`, isn't it better to stick to that as the standard instead of changing the huge number of lines to `string.`?